features/promotion/tracking_usage_limit/decreasing_promotion_usage_after_cancelling_order.feature

```diff
@@ -0,0 +1,22 @@
+@managing_promotions
+Feature: Decreasing a promotion usage after cancelling an order
+    In order to properly limit promotion usage
+    As an Administrator
+    I want to have a promotion usage limit decreased after order cancellation
```

Comment:
This sentence is really misleading form me. `Decreased usage limit` does not equal `Decreased usage`. Limit is still same and the promotion is not used, so we should decreased usage counter, not limit.